common/index.ts

```diff
@@ -0,0 +1,4 @@
+export const PLUGIN_ID = 'opendistroKibanaReports';
+export const PLUGIN_NAME = 'opendistro_kibana_reports';
+
+export const API_PREFIX = '/api/v1/reporting';
```

Comment:
add v1 to api endpoint is tricky, would suggest just use /api/reporting

Reply:
make sense